index.html

```diff
@@ -243,15 +217,15 @@
         <p class="note">
           Support for <code>hidden</code> attribute is maintained for
           historical reasons. Developers should use
-          <code>visibilityState</code> where possible.
+          <code><a data-link-for="Document">visibilityState</a></code> where possible.
         </p>
       </section>
       <section>
         <h3>
-          <code>visibilityState</code> attribute
+          <code><a data-link-for="Document">visibilityState</a></code> attribute
```

Comment:
Probably move `data-link-for` to the section, if this section just deals with Document.

Reply:
This link points to the VisibilityState enum if we remove `data-link-for`...  but other links of the visibilityState attribute can work without data-link-for.